.travis/run-tests.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Upgrade PHPUnit to 6+ if PHP version is 7+.
+if [ $(php -v | head -n 1 | cut -d ' ' -f 2 | cut -d '.' -f 1) -ge 7 ]; then
+  composer run-script drupal-phpunit-upgrade;
+fi
```

Comment:
Upgrade PHPUnit to 6+ if PHP version is 7+, because otherwise they are incompatible.